docs/metrics-analysis-prometheus/collected-reported-metrics.md

```diff
@@ -0,0 +1,117 @@
+# Collected and reported metrics
+
+The idea behind this repository is to gather as much information as
+possible from the requests sent by GitHub via the Webhook.
+
+In first place, it is important to differentiate the `workflow_run`
+and the `workflow_job` API requests.
+
+The `workflow_run` request is triggered when a workflow run is `requested`,
+`in_progress`, or `completed`.
```

Comment:
how about cancelled and skipped? Should it be mentioned?

Reply:
Done, I also added other events for `workflow_job`